### ISSUE TITLE
Error in phenotype mapping in GEL PanelApp

### ISSUE THREAD

**Issue by prashantuniyal02** [opentargets/issues#3279]
**Describe the bug**
GEL PanelApp lists [phenotypes for IGF1](https://panelapp.genomicsengland.co.uk/panels/162/gene/IGF1/#!details) as:
- Growth retardation with deafness and mental retardation due to IGF1 deficiency, 608747
- MPD
- microcephalic primordial dwarfism

Since MPD is a synonym for [Chronic myeloproliferative disorder](https://platform.opentargets.org/disease/EFO_0002428), the platform suggests a link between chronic myeloproliforative disorders and IGF1 which does not exists.

**Expected behaviour**
The platform should not suggest a link between chronic myeloproliforative disorders and IGF1.
Work with GEL PanelApp to remove MPD as a phenotype.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to [Evidence for IGF1 in chronic myeloproliferative disorder](https://platform.opentargets.org/evidence/ENSG00000017427/EFO_0002428)
2. Scroll down to GEL PanelApp widget
3. See error

**Additional context**
Raised on the community [here](https://panelapp.genomicsengland.co.uk/panels/162/gene/IGF1/#!details).

**Comment by prashantuniyal02** [comment]
@ireneisdoomed @d0choa GEL PanelApp have updated the lists phenotypes for IGF1 as:

- Insulin-like growth factor I deficiency, OMIM:608747
- Microcephalic primordial dwarfism

MPD has been removed from the list of phenotypes for IGF1 and now the evidence between between chronic myeloproliforative disorders and IGF1 should not exist in next release.

**Comment by ireneisdoomed** [comment]
@prashantuniyal02 if you are in conversations with Panel App, please let us know when the new data is ready. Thanks!

**Comment by DSuveges** [comment]
Based on the new dataset provided at 2024.05.14, the following diseases, phenotypes are associated with IGF1:
```
IGF1  ["Insulin-like growth factor I deficiency, OMIM:608747","Microcephalic primordial dwarfism"]  162  Insulin-like growth factor I deficiency
IGF1  ["Insulin-like growth factor I deficiency, OMIM:608747","Microcephalic primordial dwarfism"]  162  Microcephalic primordial dwarfism
IGF1  ["Insulin-like growth factor I deficiency, OMIM:608747"]                                      473  Insulin-like growth factor I deficiency
IGF1  ["Insulin-like growth factor I deficiency, OMIM:608747"]                                      484  Insulin-like growth factor I deficiency
IGF1  ["Insulin-like growth factor I deficiency, OMIM:608747"]                                      131  Insulin-like growth factor I deficiency
IGF1  ["Insulin-like growth factor I deficiency, OMIM:608747"]                                      478  Insulin-like growth factor I deficiency
IGF1  ["Insulin-like growth factor I deficiency, OMIM:608747"]                                      199  Insulin-like growth factor I deficiency
IGF1  ["Insulin-like growth factor I deficiency, OMIM:608747"]                                      285  Insulin-like growth factor I deficiency
IGF1  ["Growth retardation with deafness and mental retardation due to IGF1 deficiency, 608747"]    921  Growth retardation with deafness and mental retardation due to IGF1 deficiency
```
So, the update looks good, and addresses the issue, together with ~2k increase in the number of evidence.

**Comment by prashantuniyal02** [comment]
This issue has been resolved in the latest dev version and will be fixed with the Platform 24.06 release.


![Image](https://github.com/opentargets/issues/assets/122870584/fcb280da-fb0b-45b5-a8e3-dd7063ef3745)
